api/src/main/java/org/pmiops/workbench/utils/mappers/LeonardoMapper.java

```diff
@@ -58,6 +60,18 @@ default void addCloudServiceEnum(@MappingTarget LeonardoGceConfig leonardoGceCon
     leonardoGceConfig.setCloudService(LeonardoGceConfig.CloudServiceEnum.GCE);
   }
 
+  GceWithPdConfig toGceWithPdConfig(LeonardoGceWithPdConfig leonardoGceWithPdConfig);
+
+  @Mapping(target = "cloudService", ignore = true)
+  @Mapping(target = "bootDiskSize", ignore = true)
```

Comment:
what we gona do with bootDiskSize? Just no use?

Reply:
We just pass this param to Leo here. It will use this param to set the VM's boot disk size. It's different from the data disk (PD).